deltachat-ios/AppDelegate.swift

```diff
@@ -17,7 +16,7 @@ enum ApplicationState {
 
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterDelegate {
-    private let dcContext = DcContext()
+    private let dcContext = DcContext.shared
```

Comment:
i would just create a normal object a before here, this is much simpler, more understandable and also allows other usage of `DcContext` in the future